core/src/main/java/com/linecorp/armeria/common/HttpMethod.java

```diff
@@ -139,6 +140,14 @@ public static boolean isSupported(String value) {
         return false;
     }
 
+    /**
+     * Returns the <a href="https://developer.mozilla.org/en-US/docs/Glossary/Idempotent">idempotent</a>
+     * HTTP methods.
```

Comment:
`HTTP methods - {@link #GET}, {@link #HEAD}, {@link #PUT} and {@link #DELETE}.` ?